docs/DevEnv.md

```diff
@@ -0,0 +1,66 @@
+## Netplugin Developer's Guide
+This document describes the development environment and typical steps to hack on netplugin.
+While there are alternative ways, the steps listed here is used by a lot of developers while
+making incremental changes and doing frequent compilation for a quicker development cycle.
+
+Happy hacking!
+
+### 1. Check-out a tree. `Estimated time: 2 minutes`
+Notes: Make sure GOPATH is set. This is a one time activity.
```

Comment:
I am wondering if setting GOPATH on host is really needed.

I remember this was required for running systemtests but since right now the tests are in python this might not be a requirement anymore. Is this requirement coming from anything else?